src/raster/r.damflood/SWE.c

```diff
@@ -255,11 +272,11 @@ void shallow_water(double **m_h1,double **m_u1, double **m_v1, float **m_z,float
 
 
 	//*****************************************************************************
-	// abbassamento lago (siccome c'e due volte fare poi una function)
+	// Lowering of the lake (as there is twice do then a function)
```

Comment:
I'm not sure "lowering of the lake" is the correct concept in English hydrology. In French, there would have been a direct equivalent of the Italian word that made better sense for me than my English suggestion.

Reply:
Perhaps "lake depth reduction" ??